java/ci/Dockerfile.rocky

```diff
@@ -27,13 +27,14 @@ ARG TARGETPLATFORM=linux/amd64
 # check available official arm-based docker images at https://hub.docker.com/r/nvidia/cuda/tags (OS/ARCH)
 FROM --platform=$TARGETPLATFORM nvidia/cuda:$CUDA_VERSION-devel-rockylinux$OS_RELEASE
 ARG TOOLSET_VERSION=14
+ARG CMAKE_VERSION=3.30.7
```

Comment:
Maybe update to CMake 4 if it’s not too much work?

Reply:
Currently, version 3.30.X meets our needs. Do you have a recommendation for a specific CMake 4+ version here? We can have an upgrade later, thanks